libs/light/neopixel.ts

```diff
@@ -477,11 +501,17 @@ namespace light {
         /**
          * This function is deprecated.
          */
-        //% blockId=neophoton_set_color block="%strip=variables_get|photon set pen color %color=colorWheelPicker"
+        //% blockId=neophoton_set_color block="%strip=variables_get|photon set pen hue %hue=colorWheelPicker"
+        //% help="light/set-photon-pen-hue"
         //% parts="neopixel" deprecated=1 blockHidden=true
```

Comment:
Why is this deprecated and hidden? Did you mean that?

Reply:
That's a bug.

Reply:
Fixed in https://github.com/Microsoft/pxt-common-packages/pull/133